source/plugin/permissions.rst

```diff
@@ -370,3 +370,32 @@ The ``ContextCalculator`` can be registered via:
 .. code-block:: java
     
     permissionService.registerContextCalculator(contextCalculator);
+
+For Forge Mods
+========
```

Comment:
Travis failed the build because this underline was too short (they have to match the length of the title line).